### ISSUE TITLE
Availability on OpenRepos / Warehouse

### ISSUE THREAD

**Issue by acidicX** [aebruno/whisperfish#50]
Hi,

thanks for your work! It would be nice to have a precompiled binary on OpenRepos / Warehouse, so we can test without building it on our own :-)

Also, I noticed that there is a client called SailSecure (https://github.com/geobra/harbour-sailsecure), which is already available on Warehouse. Did you have any contact? Maybe join forces on one app?

Cheers,
acidicX

**Comment by carepack** [comment]
If you look on the top of the code page you'll "releases". There you'll find the packages:
https://github.com/aebruno/whisperfish/releases

schorsch aleady commented you're thinks. Don't know who was the first with this kind of client:
https://openrepos.net/comment/13635#comment-13635

**Comment by acidicX** [comment]
> If you look on the top of the code page you'll "releases". There you'll find the packages:
https://github.com/aebruno/whisperfish/releases

Yep, but I still have to install/update manually. Warehouse makes more sense for beta testing with more users because it's more convenient.

> schorsch aleady commented you're thinks. Don't know who was the first with this kind of client:
https://openrepos.net/comment/13635#comment-13635

Ah I did not read the comments. Well, if he is thinking about it, maybe let him know via the comments that you would welcome his contributions? Maybe he could even make this available in OpenRepos, as he seems to have experience with that..?

**Comment by carepack** [comment]
The comment is not the newest and in the meantime schorsch released a new version last week. I think bruno got his reasons for not beeing on openrepos already. Quality / Features and that stuff I think.

**Comment by acidicX** [comment]
> The comment is not the newest and in the meantime schorsch released a new version last week.

To quote @geobra from his comment on OpenRepos:
>  Instead, just let us wait for whisperfish to hit Jolla Harbour or openrepos.

That obviously didn't happen in the meantime, so he probably thought to go on with his version. As I said, maybe extend a hand, show that his contributions here would be welcome.

> Quality / Features

Well, OpenRepos is for inofficial packages and beta testing, so it should not matter that much. It would certainly make it available to a broader audience, maybe even attract contributions. Anyway, maybe @aebruno could list his reasons or create a list what has to be done before this project can be made available on OpenRepos.

**Comment by d9h02f** [comment]
> OpenRepos is for inofficial packages and beta testing
No, it's just a third party where people can easily upload packages for others to easily download and install them, for people who don't want to or can't use Jolla's store for this (Jolla has a lot of restrictions that are not very important to the end user).

**Comment by acidicX** [comment]
Ok, so it's *mostly used* for it. I would still like to hear from @aebruno, maybe he just didn't have the time to publish it on OR... or something else that could be done by someone else, but without knowing, it's relying on your own guesswork.

**Comment by aebruno** [comment]
Pull requests are more than welcome! The app still needs _a lot_ of work and have been targeting users who can build the app from source thus far. I likely won't publish the app on OpenRepos as it's unlcear if Open Whisper Systems supports 3rd party clients connecting to their servers. If that's the case future builds will not point at their servers and would require running your own server infrastructure.

**Comment by d9h02f** [comment]
> it's unlcear if Open Whisper Systems supports 3rd party clients connecting to their servers

They prefer people using the official Signal clients, but they don't intend to block third party clients (which would be impossible anyway because the third party clients could just pretend to be the official client).

**Comment by marxistvegan** [comment]
@aebruno Might be worth considering to add to openrepos. Though it is mentioned as a gray area on the thread, I think it is worth moving forward. Their feedback was more to the point of whether it would state up to date with the communication. For the record https://whispersystems.discoursehosting.net/t/resources-to-support-jolla-sailfishos-devices/712/9 

I know we have discussed this in passing, but felt it is relevant as with the SailfishX there are a lot of new users.

**Comment by Caballlero** [comment]
@aebruno  will Whisperfish  -v 0.5  hit openrepos.net? will nice to close this issue too.

/Thanks
